### DISCUSSION TITLE
Prototype kit dependencies

### DISCUSSION THREAD

**Discussion by andoyle** [twilio-labs/paste#1703] [Q&A] [answered]
Is it okay for me to update to the latest version of Paste on the Prototype kit or should this be done by DSYS? It can be confusing to fork it and then use a newly updated / released component that doesn't look right or work. 

![Screenshot 2021-07-28 at 13 55 00](https://user-images.githubusercontent.com/69856100/127326286-fb393088-067e-42fe-b001-93f21d004cbc.png)

**Comment by nkrantz** [comment, marked as answer]
Hey @andoyle, Paste deps should be up to date in the [Prototyping Kit](https://codesandbox.io/s/github/twilio-labs/paste-prototype-kit) now. Thanks for the reminder!

**Reply by andoyle** [reply]
Amazing—thanks!

**Comment by andoyle** [comment]
Can we get the Prototype kit deps updated again please? And, any "unhandled errors" looked at? 🙏
![Screenshot 2023-03-13 at 10 00 11](https://user-images.githubusercontent.com/69856100/224668995-b5797b4c-3224-4967-9e8b-98e3d4e9da3b.png)

**Reply by nkrantz** [reply]
Hey @andoyle - working on this now. Thanks for the reminder!

**Reply by andoyle** [reply]
Thanks so much 🙏

**Reply by SiTaggart** [reply]
Updated. Codesandbox has some new infra, so it lives here now https://codesandbox.io/p/github/twilio-labs/paste-prototype-kit/

**Reply by andoyle** [reply]
This new infra is... interesting. I can't seem to fork the kit easily now (need to connect to GitHub) and it looks like its creating a branch instead. Not sure this is an issue, just wanted to flag it. See: https://codesandbox.io/p/github/twilio-labs/paste-prototype-kit/csb-cz4o18/draft/reverent-edison

**Reply by SiTaggart** [reply]
Yeah they are slowly moving away from sandboxes to working environments, probably so they can earn money. So forks to direct git branches makes sense in those terms for them. We'll see how we go, there isn't many alternatives out there.

**Reply by SiTaggart** [reply]
Might start looking at https://stackblitz.com

**Reply by andoyle** [reply]
Fair enough. I'll keep going with it and report back if anything starts blocking my flow 👍
